ocrd/ocrd/cli/workspace.py

```diff
@@ -583,17 +584,38 @@ def list_groups(ctx):
     print("\n".join(workspace.mets.file_groups))
 
 # ----------------------------------------------------------------------
-# ocrd workspace list-pages
+# ocrd workspace list-page
 # ----------------------------------------------------------------------
 
 @workspace_cli.command('list-page')
+@click.option('-f', '--output-format', help="Output format", type=click.Choice(['one-per-line', 'comma-separated', 'json']), default='one-per-line')
+@click.option('-D', '--chunk-number', help="Partition the return value into n roughly equally sized chunks", default=1, type=int)
+@click.option('-C', '--chunk-index', help="Output the nth chunk of results, -1 for all of them.", default=-1, type=int)
+@click.option('-r', '--page-id-range', help="Restrict the pages to those matching the provided range, based on the @ID attribute. Separate start/end with ..")
+@click.option('-R', '--numeric-range', help="Restrict the pages to those in the range, in numerical document order. Separate start/end with ..")
 @pass_workspace
-def list_pages(ctx):
+def list_pages(ctx, output_format, chunk_number, chunk_index, page_id_range, numeric_range):
     """
     List physical page IDs
     """
     workspace = Workspace(ctx.resolver, directory=ctx.directory, mets_basename=ctx.mets_basename)
-    print("\n".join(workspace.mets.physical_pages))
+    find_kwargs = {}
+    if page_id_range:
+        find_kwargs['pageId'] = page_id_range
+    ids = sorted(list(set([x.pageId for x in workspace.mets.find_files(**find_kwargs)])))
+    if numeric_range:
+        start, end = map(int, numeric_range.split('..'))
+        ids = ids[start:end]
```

Comment:
1) The lower interval limit is not included while the higher interval limit is.

```
ocrd workspace list-page -f comma-separated -R 5..10
```

Should rather return

```
PHYS_0005,PHYS_0006,PHYS_0008,PHYS_0009,PHYS_0010
```

Just like:

```
ocrd workspace list-page -f comma-separated -r PHYS_0005..PHYS_0010
```
does.

Currently, it starts from 6 because of the way indices work in most of the programming languages. However, with pages, there's not a page 0. What could help here is just subtracting 1 from the lower interval the user provides. 
```suggestion
    if numeric_range:
        start, end = map(int, numeric_range.split('..'))
        ids = ids[start-1:end]
```

This would also make the single value intervals, such as `5..5`, valid. Currently, such intervals return nothing.

It is a bit annoying to deal with such changes, but in the end, it is good for consistency. The same may or may not apply to the chunk index based on the mindset we have there. Since it is called an `index`, the 0 is still good I guess.

Reply:
You're right. As I said, I just implemented this quickly and I am prone to [off by one errors](https://en.wikipedia.org/wiki/Off-by-one_error)...